marlowe-dashboard-client/src/Pickup/Types.purs

```diff
@@ -0,0 +1,50 @@
+module Pickup.Types
+  ( State
+  , Screen(..)
+  , Card(..)
+  , Action(..)
+  ) where
+
+import Prelude
+import Analytics (class IsEvent, defaultEvent)
+import Data.Maybe (Maybe(..))
+import Marlowe.Semantics (PubKey)
+import WalletData.Types (WalletNicknameKey)
+
+type State
+  = { screen :: Screen
+    , card :: Maybe Card
+    }
+
+-- there's only one pickup screen at the moment, but we might need more, and
+-- in any case it seems clearer to specify it explicitly
```

Comment:
not sure I agree, if its not used and might never be, it's confusing

Reply:
We will need the GDPR screen, I just haven't put it in yet (or updated the comment) - next time.

Reply:
ok